iOS Gesture Alphabet/ViewController.swift

```diff
@@ -6,14 +6,53 @@
 //
 
 import UIKit
+import CoreMotion
 
 class ViewController: UIViewController {
 
+    
+    @IBOutlet weak var xAccel: UITextField!
+    @IBOutlet weak var yAccel: UITextField!
+    @IBOutlet weak var zAccel: UITextField!
+    
+    var motion = CMMotionManager()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
+        
+        myAccelerometer()
+    }
+    
+    
+    func myAccelerometer() {
```

Comment:
Make sure to add a comment indicating what this function does. Not sure if we are keeping this function for the final product, but it seems to be a great start for the test app.